app/views/purchase_or_donate_mailer/please_give_email.html.slim

```diff
@@ -0,0 +1,13 @@
+  body
+    h1 Your assisstance is needed!
+    p
+      We have valued your help over the years, and now we need to ask for your
+      help again. With a donation of $$$ you can help us continue our
```

Comment:
Looks good, but you shouldn't be worried about long lines - users' email clients will take care of that formatting.

Only add a newline when you want to add a paragraph break or similar.